core/engine/engine_test.go

```diff
@@ -179,7 +206,7 @@ func TestNotifyPublish(t *testing.T) {
 }
 
 func TestNotifyPutAndRemoveCids(t *testing.T) {
-	t.Skip("skipping test since it is flaky on the CI. See https://github.com/filecoin-project/indexer-reference-provider/issues/12")
+	// t.Skip("skipping test since it is flaky on the CI. See https://github.com/filecoin-project/indexer-reference-provider/issues/12")
```

Comment:
```suggestion
	t.Skip("skipping test since it is flaky on the CI. See https://github.com/filecoin-project/indexer-reference-provider/issues/12")
```